samza-core/src/main/java/org/apache/samza/coordinator/stream/CoordinatorStreamManager.java

```diff
@@ -129,69 +119,19 @@ public void bootstrapCoordinatorStreamConsumer() {
    */
   public Set<CoordinatorStreamMessage> getBootstrappedStream(String source) {
     if (coordinatorStreamConsumer == null) {
-      throw new UnsupportedOperationException(String.format("CoordinatorStreamConsumer is not initialized in the CoordinatorStreamManager. "
-          + "input source: %s", source));
+      throw new UnsupportedOperationException(String.format("CoordinatorStreamConsumer is not initialized in the CoordinatorStreamManager. "));
     }
     return coordinatorStreamConsumer.getBootstrappedStream(source);
   }
 
-  /**
-   * Register the coordinator stream consumer.
-   */
-  public void registerCoordinatorStreamConsumer() {
-    if (coordinatorStreamConsumer != null) {
-      log.info("Registering coordinator system stream consumer from {}.", source);
-      coordinatorStreamConsumer.register();
-    }
-  }
-
-  /**
-   * Registers the coordinator stream producer for a given source.
-   * @param source the source to register
-   */
-  public void registerCoordinatorStreamProducer(String source) {
-    if (coordinatorStreamProducer != null) {
-      log.info("Registering coordinator system stream producer from {}.", source);
-      coordinatorStreamProducer.register(source);
-    }
-  }
-
   /**
    * Returns the config for the coordinator stream consumer.
    * @return Config of the coordinator stream consumer.
    */
-  public Config getCoordinatorStreamConsumerConfig() {
+  public Config getConfig() {
```

Comment:
Why does this need to be a public method? 
Is the javadoc supposed to be "Get config **from** the coordinator stream"?